apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -13,12 +13,16 @@ import {
   filterOutBackgrounds
 } from '@cdo/apps/code-studio/assets/searchAssets';
 import experiments from '@cdo/apps/util/experiments';
-import Button from '@cdo/apps/templates/Button';
+import Button, {ButtonHeight} from '@cdo/apps/templates/Button';
 import {AnimationProps} from '@cdo/apps/p5lab/shapes';
 import {isMobileDevice} from '@cdo/apps/util/browser-detector';
 
 const MAX_SEARCH_RESULTS = 40;
 const MAX_HEIGHT = 460;
+// Represents the height of the "Animation Library" title, Search Bar, and Categories list
+const MODAL_HEADER_HEIGHT = 35 + 30 + 25;
+// Default button height plus padding
+const MODAL_FOOTER_HEIGHT = ButtonHeight.default + 15;
```

Comment:
we shouldn't be hardcoding element heights -- we can use react refs to tell us about rendered components ([example of getting height of an element](https://stackoverflow.com/questions/35153599/reactjs-get-height-of-an-element))